src/Microsoft.ML.StandardLearners/Standard/MultiClass/Ova.cs

```diff
@@ -187,11 +187,11 @@ public override MulticlassPredictionTransformer<OvaPredictor> Fit(IDataView inpu
 
                     if (i == 0)
```

Comment:
>if (i == 0) [](start = 20, length = 11)

Do we need this special case? It looks like featureColumn is only used after the whole for loop, we can pass predictors[0].FeatureColumn instead.

Reply:
I need to understand why this special case was created in the first place.  I will take a closer look into this in my upcoming investigations into OVA/PKPD 

---
In reply to: [234314724](https://github.com/dotnet/machinelearning/pull/1574#discussion_r234314724) [](ancestors = 234314724)